project/cmake/scripts/common/prepare-env.cmake

```diff
@@ -55,15 +56,15 @@ get_filename_component(prefix "${DEPENDS_PATH}" ABSOLUTE)
 set(APP_LIBDIR "${prefix}/lib")
```

Comment:
These variables can go since they don't seem to be used anymore in `kodi-config.cmake.in`.